source/ui/REIXS/REIXSSampleChamberButtonPanel.h

```diff
@@ -50,6 +52,7 @@ protected slots:
 
 	QToolButton *createQToolButton(QString text, QIcon icon);
 	AMControlMoveButton *createAMControlMoveButton(QString text, QIcon icon);
+	AMXYThetaControlMoveButton *createAMXYThetaControlMoveButton(QString text, QIcon icon);
```

Comment:
For the `text` and `icon` parameters, you should use `const QString &` and `const QIcon &`.